notebooks/features/responsible_ai/Interpretability - PDP and ICE explainer.ipynb

```diff
@@ -801,7 +801,7 @@
                 "\n",
                 "We can overlay the PDP on top of ICE plots. In the graph, the red line shows the PDP plot for the \"age\" feature, and the black lines show ICE plots for 50 randomly selected observations. \n",
                 "\n",
-                "The visualization will show that all curves follow a similar course. That means that the PDP (red line) is already a good summary of the relationships between the displayed feature \"age\" and the model's average predictions of \"income\""
+                "This is done for showing that all curves seem to follow the same course. That means that the PDP is already a good summary of the relationships between the displayed feature \"age\" and the model predictions of \"income\""
```

Comment:
Not sure what you mean by "This is done for showing that all curves seem to follow the same course." might want to revise this sentence

Reply:
I feel the original sentence makes more sense to me.